libs/s25main/figures/nofScout_Free.cpp

```diff
@@ -156,7 +156,7 @@ struct IsScoutable
 void nofScout_Free::GoToNewNode()
 {
     std::vector<MapPoint> available_points =
-      gwg->GetPointsInRadius<-1>(flag->GetPos(), SCOUT_RANGE, Identity<MapPoint>(), IsScoutable(player, *gwg));
+      gwg->GetMatchingPointsInRadius<-1>(flag->GetPos(), SCOUT_RANGE, IsScoutable(player, *gwg));
```

Comment:
no test for this?

Reply:
No not yet. And IMO out of scope for this PR. Adding tests for the figures/jobs is quite complicated as you need to set up a whole world etc. Might do later but not in this PR. I think it is clear from the change that it doesn't change behavior if the function works as expected and that at least is tested

Reply:
Test added